### ISSUE TITLE
explode() expects parameter 2 to be string, null given

### ISSUE THREAD

**Issue by akroii** [fritzmg/contao-inherit-article#9]
Hallo fritzmg,
die Erweiterung wirft einen Fehler wenn man direkt den Frontend aufruft.

C4.8.4
contao-inherit-article 2.1.3

Debug-Mode:
![Bildschirmfoto 2019-11-06 um 09 10 14](https://user-images.githubusercontent.com/8830861/68280284-83c32780-0075-11ea-9136-44a1f83a0b19.png)

```
[2019-11-06 09:09:12] request.CRITICAL: Uncaught PHP Exception TypeError: "explode() expects parameter 2 to be string, null given" at /Applications/MAMP/htdocs/tps/vendor/fritzmg/contao-inherit-article/src/EventListener/InheritArticleListener.php line 251 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): explode() expects parameter 2 to be string, null given at /Applications/MAMP/htdocs/tps/vendor/fritzmg/contao-inherit-article/src/EventListener/InheritArticleListener.php:251)"} []
```

**Comment by fritzmg** [comment]
Please post Stack Traces as Text, not as images. Simply click on the tab "Stack Trace" and copy the text of the exception Stack Trace.

**Comment by akroii** [comment]
```
TypeError:
explode() expects parameter 2 to be string, null given

  at vendor/fritzmg/contao-inherit-article/src/EventListener/InheritArticleListener.php:251
  at explode('_', null)
     (vendor/fritzmg/contao-inherit-article/src/EventListener/InheritArticleListener.php:251)
  at InheritArticleBundle\EventListener\InheritArticleListener->applyThemeAndLayout(object(PageModel), object(LayoutModel))
     (vendor/fritzmg/contao-inherit-article/src/EventListener/InheritArticleListener.php:78)
  at InheritArticleBundle\EventListener\InheritArticleListener->onGetPageLayout(object(PageModel), object(LayoutModel), object(PageRegular))
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:252)
  at Contao\PageRegular->getPageLayout(object(PageModel))
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:77)
  at Contao\PageRegular->prepare(object(PageModel))
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:48)
  at Contao\PageRegular->getResponse(object(PageModel), true)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/FrontendIndex.php:343)
  at Contao\FrontendIndex->renderPage(object(PageModel))
     (vendor/symfony/http-kernel/HttpKernel.php:151)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:198)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/index.php:31)
```

**Comment by fritzmg** [comment]
Please try `dev-master` and report back if this problem is fixed there.

**Comment by akroii** [comment]
Yep, bug is gone.

**Comment by fritzmg** [comment]
Released as `2.1.4`. Somebody reported this to me a long time ago, but never gave feedback whether the current master branch fixed the problem.
